web/ui/templates/targets.html

```diff
@@ -1,57 +1,72 @@
-{{define "head"}}<!-- nix -->{{end}}
+{{define "head"}}
+<link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/css/targets.css?v={{ buildVersion }}">
```

Comment:
I see, thanks for the info. Not sure the overhead is worth it, but let's keep it consistent for now.